### PR TITLE
Add Adapters for EcoDeFi

### PR DESCRIPTION
**Pull request by alexchang2022**
##### Twitter Link:
https://twitter.com/esg_financial/

##### List of audit links if any:
https://www.certik.com/projects/esg-financial

##### Website Link:
https://ecodefi.net/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://ecodefi.net/img/img_logo_esg.7e50b672.png

##### Current TVL:
618,439 USD
##### Chain:
Binance Smart Chain(BSC)

##### Short Description (to be shown on DefiLlama):
The ESG Protocol is an Binance smart contract for supplying or borrowing assets. Through the eToken contracts, accounts on the blockchain supply capital (BNB or BEP-20 tokens) to receive eTokens or borrow assets from the protocol (holding other assets as collateral). The ESG eToken contracts track these balances and algorithmically set interest rates for borrowers.

##### Token address and ticker if any:
0x0985205D53D575CB07Dd4Fba216034dc614eab55

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Lending

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
Chainlink

##### forkedFrom (Does your project originate from another project):
Compound

##### methodology (what is being counted as tvl, how is tvl being calculated):
Sum of total supply of underlying assets in every eTokens contract.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/ecodefi exports TVL: 
        
 
 
    bsc-staking               11.11 k
    bsc                       612.36 k
    staking                   11.11 k
    
    total                    612.36 k

File: projects/ecodefi/index.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  let staked = await getESGStakingValue('bsc', chainBlocks['bsc']);
+  return toUSDTBalances(staked);
+}
+
+module.exports = {
+  timetravel: true,
+  misrepresentedTokens: false,
+  methodology: 'counts the number of Total value locked in ESG protocol.',
+  bsc: {
+	  tvl: bsc,
```
Comment (g1nt0ki) [inline projects/ecodefi/index.js:173]: the staking part looks good, can you update the tvl computation part ot below, we have a helper that makes it easy to compute both deposits and borrowed amount:

    const { compoundExports } = require('../helper/compound');


    module.exports = {
      bsc: {
        ...compoundExports('0xfd1f241ba25b8966a14865cb22a4ea3d24c92451', 'bsc'),
        staking,
      },
    };

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by alexchang2022** [comment]
Updated as required.
Please review it, thanks.

**Comment by llamatester** [comment]
The adapter at projects/ecodefi exports TVL: 
        
 
 
    bsc                       541.51 k
    bsc-staking               907.6210268159999
    bsc-borrowed              54.91 k
    staking                   907.6210268159999
    borrowed                  54.91 k
    
    total                    541.51 k

**Pull request merged**